compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -250,7 +271,7 @@ void Interpreter::interpret()
     kernel->execute();
```

Comment:
So, new classes are not used in this PR?

Reply:
It is a horrible mistake, thank you.

Reply:
Fixed